query/src/sql/sql_parser.rs

```diff
@@ -163,7 +163,20 @@ impl<'a> DfParser<'a> {
                         self.parser.next_token();
 
                         if self.consume_token("TABLES") {
-                            Ok(DfStatement::ShowTables(DfShowTables))
+                            let tok = self.parser.next_token();
+                            match &tok {
+                                Token::EOF => Ok(DfStatement::ShowTables(DfShowTables::All)),
```

Comment:
What happens to the following SQL?
``` SQL
SHOW TABLES  ; 
SHOW TABLES -- comment
```

Reply:
sorry, I will add some tests and check if it's ok tonight.